src/examples/modal/ModalDanger.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Modal, Header, Body, Footer, FooterItem, Close } from '@zendeskgarden/react-modals';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <Row>
+      <Col textAlign="center">
+        <Button onClick={() => setVisible(true)} isDanger>
+          Open danger modal
+        </Button>
+        {visible && (
+          <Modal onClose={() => setVisible(false)}>
+            <Header isDanger>Remove plant food from your garden</Header>
+            <Body>
+              Plant food is great for the health of a garden. Are you sure you want to remove it?
```

Comment:
We don't ask users if they are sure in Zendesk. Per our tone guidelines, we tell people what the effects of a destructive action are. Perhaps something more like

> Plant food helps gardens grow. Removing plant food negatively affects the plant health.

@vsrixyz what do you think?

Reply:
Made these changes for now to keep this moving.